buildpack.toml

```diff
@@ -63,6 +69,20 @@ api = "0.7"
       type = "Apache-2.0"
       uri = "https://github.com/spring-cloud/spring-cloud-bindings/blob/main/LICENSE"
 
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:vmware:spring_cloud_bindings:2.0.0:*:*:*:*:*:*:*"]
+    id = "spring-cloud-bindings"
+    name = "Spring Cloud Bindings"
+    purl = "pkg:generic/springframework/spring-cloud-bindings@2.0.0"
+    sha256 = ""
+    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
+    uri = "https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-bindings/2.0.0/spring-cloud-bindings-2.0.0.jar"
```

Comment:
it kinda feels odd to me to set a specific version of the library here.

Shouldn't we have the 2 versions set as dependencies?